app/test/shared/email_test.dart

```diff
@@ -135,4 +135,35 @@ void main() {
           'Pub Site Admin\n');
     });
   });
+
+  group('Uploaders', () {
+    test('uploader verification', () {
+      final message = createUploaderVerificationEmail(
+        packageName: 'pkg_foo',
+        activeAccountEmail: 'active@example.com',
+        addedUploaderEmail: 'uploader@example.com',
+        verificationUrl:
+            'https://pub.dartlang.org/verification/add-uploader/abcdef1234567890',
+      );
+      expect(message.from.toString(), 'Pub Site Admin <pub@dartlang.org>');
+      expect(message.recipients.map((e) => e.toString()).toList(),
+          ['uploader@example.com']);
+      expect(message.subject, 'Pub invitation to collaborate: pkg_foo');
+      expect(
+          message.bodyText,
+          'Dear package maintainer,\n'
```

Comment:
Do we really want to maintain tests like this?

Wouldn't be easier to just verify that `bodyText` and `subject` contains `pkg_foo` and `abcdef1234567890`, as sanity checks. This other things is just going to break whenever someone changes the email.

Reply:
The point of such tests (both in pana and on the pub site, e.g. golden files) is that we are aware of the extent of the change. It also help us to double-check the results if the templates go more complex.

Reply:
I made a small function to maintain goldens in a file: 
https://github.com/dart-lang/dart-protoc-plugin/blob/master/test/golden_file.dart
To update the file, just delete it and run again.